### PR TITLE
FBFW-127 - Setup Redux for messaging

### PR DESCRIPTION
**Pull request by mariomelchor**
- Setting up redux state for messaging.
- Download PR Confirm seeing messages in redux when loading beta

Screen Shot 2022-04-28 at 6 14 35 PM

### CODE REVIEW

File: beta-src/src/state/game/game-api-slice.ts [hunk 1]
```diff
@@ -708,6 +727,12 @@ const gameApiSlice = createSlice({
             }
           });
         }
+      })
+      // Fetch Game Messages
+      .addCase(fetchGameMessages.fulfilled, (state, action) => {
+        if (action.payload) {
+          state.messages = action.payload;
```
Comment (bryancodazen) [inline beta-src/src/state/game/game-api-slice.ts:734]: Question: isn't the payload here going to contain meta data like `msg`, `success`, etc? If so don't we need to assign the `data`, not the entire payload.

**Review by bryancodazen — APPROVED** [review]
LGTM, left a comment with a question.

**Review by justinyoocodazen — APPROVED** [review]

**Comment by mariomelchor** [comment]
> 
@bryancodazen 
Currently it only pulls the global messages on load. Would this be enough or do we have to pull down all messages for the logged in user Global/Notes/Messages to other countries as well?

**Comment by bryancodazen** [comment]
> @bryancodazen Currently it only pulls the global messages on load. Would this be enough or do we have to pull down all messages for the logged in user Global/Notes/Messages to other countries as well?

Yes we would need to pull all messages on initial load.

**Pull request merged**